epidataanalysis.lpr

```diff
@@ -12,7 +12,7 @@
   Interfaces, sysutils, Forms,
 
   // Required EpiData units
-  ana_procs, wizard_form, datamodule, main,
+  ana_procs, wizard_form, datamodule, main, chart_options,
```

Comment:
chart_options should automatically be included in other units and is technically not needed here. You are free to decide if you want to leave the unit in or not.

Reply:
removed